Goal: Fix a typo in configure.in

Thanks to Mike Frysinger <vapier@gentoo.org>

Index: shadow-4.0.13/configure.in
===================================================================
--- shadow-4.0.13.orig/configure.in	2005-10-12 00:47:01.000000000 +0200
+++ shadow-4.0.13/configure.in	2005-10-12 01:20:48.000000000 +0200
@@ -170,7 +170,7 @@
 
 AC_CACHE_CHECK([location of faillog/lastlog/wtmp], shadow_cv_logdir,
 [for shadow_cv_logdir in /var/log /var/adm /usr/adm /etc; do
-	if test -d $logdir; then
+	if test -d $shadow_cv_logdir; then
 		break
 	fi
 done])
